docs/query-params.md

```diff
@@ -83,14 +83,27 @@ Specify an alternate style for Firefox Accounts.
 #### When to specify
 When authenticating a user with OAuth and an alternate style is needed.
 
-## Firefox/Sync parameters
-
 #### When to specify
```

Comment:
Crap. The "Firefox/Sync parameters" title was in the wrong space and I didn't look down a line. Updating. Thanks for catching that!